test/coreSpec.js

```diff
@@ -1,8 +1,8 @@
-describe('Prelim Checks: ', function(){
-    it('Granim is defined', function(){
+describe('Prelim Checks: ', function () {
```

Comment:
yep sorry it's the auto indentation of my editor, maybe it could be a good idea to implement this to avoid those kind of issues: https://editorconfig.org/